api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -475,11 +480,11 @@ public static void startup(Properties p) {
 			val = OpenmrsConstants.DATABASE_NAME;
 		}
 		OpenmrsConstants.DATABASE_BUSINESS_NAME = val;
+
```

Comment:
Did you intentionally reverse setupLogAppenders() and applyLogLevels()?

Reply:
Yes. Initially, this was part of getting the settings to override the XML file, it's not strictly-speaking necessary.